documentation/en/chapter_2.md

```diff
@@ -0,0 +1,55 @@
+# 2. What is Locha Mesh?
+
+Locha Mesh is an open source project of networks and wireless resilient communications in order to build a free data transmission, with a incorruptible focus in the security and the privacy of the information flowing in the network. A network created and managed by its users. Locha Mesh is a distribuited network that doesn't belong to anyone in particular but to us all. Locha Mesh offers free and permissionless access to the network. Anyone can join it at any time, and use it to find its recipient inside the mesh, finding different paths to send and receive data packets foreseeing failures in the conection.
+
+## 2.1 Why a Mesh?
+
+This type of network suits the need to connect nodes that are constantly moving around, due this the topology is a dynamic one, diminishing the points of failure and the dependency on fixed infrastructure, to preclude the censorship or the manipulation of the mesh.
+
+The concepts of a wireless network or ad-hoc has been studied for a long time, creating diverse protocols that with more or less success deal with their biggest enemy, the _single-hop_ or is there life beyond my neighbors?
+
+Today, its use is very popular and has promoted an emergent field which offers a vast amount of applications that needs to be interconnected at all times wirelessly, interacting efficiently with eachother without having to give up the security in the communication.
+
+These networks can proportionate a safe and reliable communication system, where no person, or business can censor, veto or access any information unless it is the receipient of it. Another interesting application if the disaster hit countries, poor infrastructure, or those suffering the effects of war, pollitical repression, and its also replicable for those who value their personal information.
+
+## 2.2 Why use 915 MHz as base frecuency?
+
+Radio frecuency (RF) communication is a basic feature of a Wireless Sensor Network (WSN), its clearly and advantage in front of, for example, infra red transmission too. The use of wireless devices is regulated worldwide, most countries have a space in their radio spectrum for unlicensed use, meaning, it doesn't need a special permit for each application.
+
+Most commercial products operate in these free bands, also known as ISM (Industrial, Scientific and Medical) to avoid the cost of the licenses. Because of this, a big number of wireless technologies like ZigBee, Bluetooth, Wi-Fi, and wireless telephony among others, use these frecuencies.
```

Comment:
```suggestion
Most commercial products operate in these free bands, also known as ISM (Industrial, Scientific and Medical) to avoid the cost of the licenses. Because of this, a big number of wireless technologies like ZigBee, Bluetooth, Wi-Fi, and wireless telephony among others, use these frequencies.
```